stdlib/universe/quantile_test.go

```diff
@@ -246,7 +203,7 @@ func TestQuantile_NewQuery(t *testing.T) {
 		{
 			Name:    "aggregate with column",
 			Raw:     `from(bucket:"testdb") |> range(start: -1h) |> quantile(q: 0.99, method: "estimate_tdigest", column: "1")`,
-			WantErr: true,
+			WantErr: false,
```

Comment:
This should be like the test you deleted above, so something like `with custom column` and an expected result.